packages/main/src/Popover.js

```diff
@@ -420,7 +365,6 @@ class Popover extends UI5Element {
 
 	shouldCloseDueOverflow(placement, openerRect) {
 		const threshold = 32;
-
```

Comment:
shouldCloseDueOverflow => shouldCloseDueToOverflow would be more accurate, but I don't mind